WordPress/Classes/ViewRelated/NUX/NUXSubmitButton.swift

```diff
@@ -0,0 +1,120 @@
+import UIKit
+import WordPressShared
+
+let NUXSubmitButtonDisabledAlpha = CGFloat(0.25)
+
+/// A stylized button used by NUX controllers. The button presents white text 
+/// surrounded by a white border.  It also can display a `UIActivityIndicatorView`.
+///
+@objc class NUXSubmitButton : UIButton
```

Comment:
This looks similar in functionality to `PurchaseButton`/`RoundedButton` although using a regular `UIActivityIndicatorView` instead of `MRActivityIndicatorView`. Maybe we could combine them and have the subclasses deal with just the styling.